rest/config_legacy.go

```diff
@@ -112,6 +115,9 @@ func (lc *LegacyServerConfig) ToStartupConfig() (*StartupConfig, DbConfigMap, er
 			Pretty:             lc.Pretty,
 			CompressResponses:  lc.CompressResponses,
 			HideProductVersion: lc.HideProductVersion,
+			HTTPS: HTTPSConfig{
+				UseTLSClient: lc.UseTLSClient,
```

Comment:
What is this going to end up being if its not specified in the config? Defaults to false maybe before flag / Default overrides?

Reply:
Tested it and the default of true does get applied if the user has not specified it in their config.